lizmap/modules/lizmap/lib/Request/Proxy.php

```diff
@@ -675,13 +682,18 @@ public static function clearLayerCache($repository, $project, $layer)
                     }
                 }
                 closedir($handle);
+                if (count($results) === 0) {
+                    return false;
```

Comment:
Since we want to remove cache, having no existing cache is not a failure IMHO. It just says that we don't have to remove anything. So you should return true here.

you should return false if opendir fails for example.